doc/release-notes/8309-postgres-required.md

```diff
@@ -0,0 +1,37 @@
+## Notes for Dataverse Installation Administrators
+
+### PostgreSQL Version 10+ Required
+
+If you are still using PostgreSQL 9.X, now is the time to upgrade. PostgreSQL is now EOL (no longer supported, as of January 2022), and the PostgreSQL driver in the next release of the Dataverse Software will no longer work with versions prior to 10. 
```

Comment:
The PostgreSQL driver, or the FlyWay thing-a-ma-bob? (or both?)

Reply:
Did I get this wrong? I thought it was on account of Flyway, but the actual requirement was the driver... no? 
Will fix!

Reply:
I think the Postgres driver is intentionally as backwards-compatible as possible; I understood that Flyway dropped 9.n when they added support for 14. A very minor point but... it might come up?

Reply:
OK, corrected - thank you.